kuma/javascript/src/signin-link.jsx

```diff
@@ -0,0 +1,70 @@
+//@flow
+import * as React from 'react';
+import { useContext } from 'react';
+
+import { getLocale, gettext } from './l10n.js';
+import GAProvider from './ga-provider.jsx';
+
+type Props = {
+    className?: string,
+    text?: string,
+};
+
+export default function SignInLink({
+    className,
+    text = gettext('Sign in'),
+}: Props): React.Node {
+    const ga = useContext(GAProvider.context);
+    const locale = getLocale();
+
+    if (typeof window === 'undefined') {
+        return null;
+    }
+
+    const LOCATION = window.location.pathname;
+    /**
+     * Send a signal to GA when a user clicks on the Sing In
+     * lnk in the header.
+     * @param {Object} event - The event object that was triggered
```

Comment:
A holdover from a previous version of the function?

Reply:
I imagine we still want to track the sign in click events, but maybe add a different label to differentiate between the header sign in link and the subscriptions management page. I will check with Kadir on this.

Reply:
I also ask seeing that the function does not currently accept any arguments.

Reply:
Kadir responded and said we can keep the same click events.